includes/definitions/discovery/freshtomato.yaml

```diff
@@ -0,0 +1,5 @@
+modules:
+    os:
+        hardware: '.1.3.6.1.4.1.8072.1.3.2.2.1.3.6.100.101.118.105.99.101'
```

Comment:
This OID is not in the snmprec... you sure the device provides it?

Reply:
Going in to LibreNMS -> Capture -> SNMP
This is the result: https://gist.github.com/FingerlessGlov3s/4479dceb23985e0b6220e4c9360ba6ee

Also just logged in this morning and show the hardware is showing up!!! Didn't change anything since the PR.
Only thing that had happened its gone from 1.67 to 1.68 last night.
Guessing 1.68 had a fix or something? 🤔

SNMPREC its now there and in the test data, now I've redone the collection.

Right I'll commit the new test data :-)